edn/src/types.rs

```diff
@@ -143,6 +162,16 @@ impl Value {
     def_as!(as_list, List, LinkedList<Value>);
     def_as!(as_set, Set, BTreeSet<Value>);
     def_as!(as_map, Map, BTreeMap<Value, Value>);
+
+    pub fn from_bigint(src: &str) -> Result<Value, NewValueError> {
```

Comment:
(I know none of this is *really* that important, I'm just trying to better understand things)